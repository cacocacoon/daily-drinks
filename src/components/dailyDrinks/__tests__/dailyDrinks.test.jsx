import React from 'react'
import TestRenderer from 'react-test-renderer'
import { Provider } from 'react-redux'
import configureMockStore from 'redux-mock-store'

import DailyDrinks from '../dailyDrinks'
import { isTSAnyKeyword } from '@babel/types'

function TestDailyDrinks({ store }) {
    return (
        <Provider store={store}>
            <DailyDrinks />
        </Provider>
    )
}

const mockStore = configureMockStore()

describe('Daily drinks', () => {
    it('show list', () => {
        const store = mockStore({
            ui: { showList: true },
            orders: { list: [] }
        })
        const testDailyDrinks = TestRenderer.create(
            <TestDailyDrinks store={store} />
        )
        expect(testDailyDrinks.toJSON()).toMatchSnapshot()
    })
})