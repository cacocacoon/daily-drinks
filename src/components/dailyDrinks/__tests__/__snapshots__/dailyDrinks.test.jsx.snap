// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Daily drinks show list 1`] = `
<div
  className="daily-drinks"
>
  <h1
    className="daily-drinks-h1"
  >
    Daily Drinks
  </h1>
  <button
    className="delete-order-btn btn"
    disabled={true}
    onClick={[Function]}
  >
    - Delete
  </button>
  <a
    href="/order"
    onClick={[Function]}
  >
    <button
      className="new-order-btn btn"
    >
      + Add
    </button>
  </a>
  <div
    className="daily-drinks-body"
  >
    <div
      className="empty-order-list"
    >
      Empty
    </div>
  </div>
</div>
`;
